DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -51,7 +51,9 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
                                  OR dbo.CheckCustomisationSectionHasLearning(Customisations.CustomisationID, Sections.SectionID) = 0
                             THEN 0
                             ELSE CAST(SUM(aspProgress.TutStat) * 100 AS FLOAT) / (COUNT(Tutorials.TutorialID) * 2)
-                         END) AS PercentComplete
+                         END) AS PercentComplete,
+                         SUM(aspProgress.TutTime) as TimeMins,

```

Comment:
I assume this returns something sensible if there's no progress for the candidate? 🙂

Reply:
It currently returns null, I'll change it so that it defaults to 0.